src/helpers/movies.js

```diff
@@ -1,15 +1,15 @@
 import fetch from "node-fetch";
 
-const API_URL = "https://plankton-app-xhkom.ondigitalocean.app/api/movies";
+const API = "https://plankton-app-xhkom.ondigitalocean.app/api/movies";
```

Comment:
Maybe we could have a better name for this variable? So it's clear what it is. We will have more variables with other URL later also so it would be good to have at better name.